qualtrics_link/util.py

```diff
@@ -226,10 +226,13 @@ def filter_person_list(person_list):
             employee_list.append(person)
 
     # If more than one Person with Employee role, check if any have prime indicator set to 'Y'
+    # Else return the first employee record
     if len(employee_list) > 0:
```

Comment:
Well, if rec1 is employee with prime_role not set, and rec2 is student with prime_role set, we will get res 2 with teh logic above. we want to return  rec1.